Recipe.md

```diff
@@ -0,0 +1,340 @@
+
+    4.  Use Finder and drag `__PROJECT_NAME__`.xcodeproj into the workspace in Xcode
+
+    5.  Use Finder and drag iOS Example.xcodeproj into the workspace in Xcode (make this below the other one, be sure you do NOT make it subordinated)
+
+    6.  Click iOS Example on the left
+
+    7.  Click Build Phases in the middle
+
+    8.  Under Link Binaries With Libraries click the plus icon, select `__PROJECT_NAME__`.framework, and then click "Add"
+
+5.  Remove identifying parts of your project
+
+    1.  Use Terminal.app to find and replace all occurrences of hard-coded strings with template variables
+
+            find -E ~/Desktop/__PROJECT_NAME__ \
+                -regex '.*\.(h|swift)' -exec sed -i '' -E -e '
+                    s-(// +Created by ).*( on ).*\.-\1__AUTHOR NAME__\2__TODAYS_DATE__.-
+                    s-(// +Copyright © ).*-\1__TODAYS_YEAR__ __ORGANIZATION NAME__. All rights reserved.-' \
+                '{}' \;
+
+
+    2.  Use Terminal.app to remove all references to development team IDs
+
+            find ~/Desktop/__PROJECT_NAME__ -name project.pbxproj \
+                -exec sed -i '' -E -e '/DevelopmentTeam = /d
+                    s/(DEVELOPMENT_TEAM = )[^;]+/\1""/' '{}' \;
+
+6.  Use Terminal.app to add additional files to the project
+
+        cd ~/Desktop/__PROJECT_NAME__/
+        curl 'https://raw.githubusercontent.com/github/gitignore/master/Swift.gitignore' -o .gitignore
+        echo '__PROJECT_NAME__.framework.zip' >> .gitignore
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/.travis.yml' -o .travis.yml
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/LICENSE' -o LICENSE
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/README.md' -o README.md
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/CHANGELOG.md' -o CHANGELOG.md
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/CONTRIBUTING.md' -o CONTRIBUTING.md
+        echo '3.0' > .swift-version
+        # Reference https://github.com/Alamofire/Alamofire/blob/master/.swift-version
+        # Reference https://github.com/CocoaPods/CocoaPods/pull/5841
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/Tests/CheckCocoaPodsQualityIndexes.rb' -o Tests/CheckCocoaPodsQualityIndexes.rb
+        curl 'https://raw.githubusercontent.com/fulldecent/swift3-module-template/master/__PROJECT_NAME__/Project.swift' -o Project.swift
+
+
+## Taste testing
+
+1.  Open `__PROJECT_NAME__`.xcworkspace
+
+2.  Use the scheme navigator to select iOS Example and the latest iPhone version
+
+3.  Select Product -> Run
+
+You should see a big white king. That means it worked!
```

Comment:
It'd be good to keep the ability to edit your README/Podspec inside Xcode too:

E.g.
<img width="280" alt="screen shot 2017-06-25 at 08 00 18" src="https://user-images.githubusercontent.com/49038/27515909-866b61ac-597c-11e7-985a-86769a9865ad.png">

( Current )
<img width="426" alt="screen shot 2017-06-25 at 07 59 54" src="https://user-images.githubusercontent.com/49038/27515910-87bc359a-597c-11e7-8065-09abedf8ae4c.png">